internal/codespaces/ssh.go

```diff
@@ -38,18 +38,35 @@ func Shell(ctx context.Context, p printer, sshArgs []string, port int, destinati
 func Copy(ctx context.Context, scpArgs []string, port int, destination string) error {
 	// Beware: invalid syntax causes scp to exit 1 with
 	// no error message, so don't let that happen.
-	cmd := exec.CommandContext(ctx, "scp",
+	connArgs := []string{
 		"-P", strconv.Itoa(port),
 		"-o", "NoHostAuthenticationForLocalhost=yes",
 		"-C", // compression
-	)
-	for _, arg := range scpArgs {
-		// Replace "remote:" prefix with (e.g.) "root@localhost:".
-		if rest := strings.TrimPrefix(arg, "remote:"); rest != arg {
-			arg = destination + ":" + rest
+	}
+
+	cmdArgs, command, err := parseSCPArgs(scpArgs)
+	if err != nil {
+		return err
+	}
+
+	cmdArgs = append(cmdArgs, connArgs...)
+
+	if len(command) > 0 {
+		cmdArgs = append(cmdArgs, "--")
```

Comment:
Should we update the comment on top of the `Copy` function? It seems the behavior has slightly changed here.